packages/models-library/tests/test_service_settings_labels.py

```diff
@@ -147,3 +147,32 @@ def test_raises_error_wrong_restart_policy() -> None:
 
     with pytest.raises(ValueError):
         SimcoreServiceLabels(**simcore_service_labels)
+
+
+def test_path_mappings_label_unsupported_size_constraints():
+    with pytest.raises(ValidationError) as exec_into:
+        PathMappingsLabel.parse_obj(
+            {
+                "outputs_path": "/so",
```

Comment:
minor: maybe using the faker for the path, or something like "/a/valid/path" is more helpful to understand the path is valid